src/Sulu/Bundle/WebsiteBundle/Resolver/ParameterResolver.php

```diff
@@ -115,6 +118,10 @@ public function resolve(
                 'locale' => $locale,
                 'url' => $url,
             ];
+
+            if ($country = $localization->getCountry()) {
```

Comment:
I have set optional if exists. It should be always present with null value?

Reply:
think i would just always set it in the assignment above. or do you see any problems with that?

Reply:
No, there is no problem, i think it's better. I'll change it